packages/extension/:w

```diff
@@ -0,0 +1,48 @@
+const fetchOutputNotes = ({
```

Comment:
I'm assuming this file was created in error while trying to save and exit out of vim? :)

Reply:
Oops!